commit f6ba5f6bc59690cb91ede18aacb1bc2ac392a738 (refs/changes/11/2211/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-08-24T14:17:19-07:00 (2 years, 1 month ago)
    
    TOOLS-1807 node updates for july 2017 node.js security releases
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/node.js/Makefile b/node.js/Makefile
index 9511c36..e8edf57 100644
--- a/node.js/Makefile
+++ b/node.js/Makefile
@@ -52,6 +52,7 @@ endif
 
 AUTOCONF_OPTS += \
 	--with-dtrace \
+	--without-snapshot \
 	--openssl-use-sys \
 	--openssl-libpath=$(DESTDIR)/lib \
 	--openssl-includes=$(DESTDIR)/usr/include \
